exercises/bob/README.md

```diff
@@ -13,6 +13,8 @@ anything.
 
 He answers 'Whatever.' to anything else.
 
+Bob's conversational partner is a purist when it comes to written communication and always follows normal rules regarding sentence punctuation in English.
```

Comment:
This has already been updated by https://github.com/exercism/rust/pull/597. Please remove this change from this PR.